From 0023e1a3d5bebe524d83111d3b8b9586a1b0587f Mon Sep 17 00:00:00 2001
From: Peter Jung <admin@ptr1337.dev>
Date: Sat, 31 Jan 2026 13:14:35 +0100
Subject: [PATCH 077/100] Revert "drm/amd/display: Use bigger VRR range if
 found in AMD vsdb"

This reverts commit ae12ee5de5a99102e949b88f19f2ff48f967593a.
---
 .../gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c    | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
index 18ddcfbcf..30455a7b1 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
@@ -12969,18 +12969,6 @@ static bool copy_range_to_amdgpu_connector(struct drm_connector *conn)
 	return is_freesync_capable(range);
 }
 
-/*
- * Returns true if range from AMD vsdb is bigger
- */
-static bool compare_ranges(struct drm_connector *conn,
-			   struct amdgpu_hdmi_vsdb_info *vsdb)
-{
-	struct drm_monitor_range_info *range = &conn->display_info.monitor_range;
-
-	return (vsdb->max_refresh_rate_hz - vsdb->min_refresh_rate_hz) >
-	       (range->max_vfreq - range->min_vfreq);
-}
-
 /**
  * amdgpu_dm_update_freesync_caps - Update Freesync capabilities
  *
@@ -13054,10 +13042,6 @@ void amdgpu_dm_update_freesync_caps(struct drm_connector *connector,
 		if (valid_vsdb_cea && is_monitor_range_invalid(connector))
 			monitor_range_from_vsdb(connector, &vsdb_info);
 
-		/* Use bigger range if found in AMD vsdb */
-		if (valid_vsdb_cea && compare_ranges(connector, &vsdb_info))
-			monitor_range_from_vsdb(connector, &vsdb_info);
-
 		if (dpcd_caps.allow_invalid_MSA_timing_param)
 			freesync_capable = copy_range_to_amdgpu_connector(connector);
 
-- 
2.52.0

